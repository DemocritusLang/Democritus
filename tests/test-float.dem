function main() int
{
  let a float;
  let b float;
  let c float;

  a = 10.0;
  b = 0.5;

  print_float(a + b);
  print_float(a - b);
  print_float(a * b);
  print_float(a / b);  

  return 0;
}
