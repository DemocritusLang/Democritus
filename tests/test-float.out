10.500000
9.500000
5.000000
20.000000
